v22.1/backup-architecture.md

```diff
@@ -0,0 +1,93 @@
+- Calculate the keys and time range (if incremental) of the backup data.
+- Determine the [leaseholder](architecture/overview.html#architecture-leaseholder) nodes for the keys to back up. 
+- Provide a plan to the notes that will execute the data export (typically the leaseholder node).
+
+To map out the storage location's directory to which the nodes will write the data, the coordinator identifies the [type](backup-and-restore-overview.html#backup-and-restore-types) of backup. This determines the name of the new (or edited) directory to store the backup files in. For example, if there is an existing full backup in the target storage location, the upcoming backup will be [incremental](take-full-and-incremental-backups.html#incremental-backups) and therefore append to the full backup after any existing incremental layers discovered in it. 
+
+For more information on how CockroachDB structures backups in storage, see [Backup collections](take-full-and-incremental-backups.html#backup-collections).
+
+### Key and time range resolution
+
+In this part of the resolution phase, the coordinator will calculate all the necessary spans of keys and their time range that the cluster needs to export for this backup. It divides the key spans based on which node is the [leaseholder](architecture/overview.html#architecture-leaseholder) of the range for that key span. Every node has a SQL processor on it to process the backup plan that the coordinator will pass to it. Typically, it is the backup SQL processor on the leaseholder node for the key span that will complete the export work.  
+
+Each of the node's backup SQL processors are responsible for:
+
+1. Asking the [storage layer](architecture/storage-layer.html) for the content of each key span.
+1. Receiving the content from the storage layer. 
+1. Writing it to the backup storage location or [locality-specific location](take-and-restore-locality-aware-backups.html) (whichever locality best matches the node). 
+
+Since any node in a cluster can become the coordinator and all nodes could be responsible for exporting data during a backup, it is necessary that all nodes can connect to the backup storage location.
+
+## Export phase 
+
+Once the coordinator has provided a plan to each of the backup SQL processors that specifies the backup data, the distributed export of the backup data begins.
+
+In the following diagram, **Node 2** and **Node 3** contain the leaseholders for the **R1** and **R2** [ranges](architecture/overview.html#architecture-range). Therefore, in this example backup job, the backup data will be exported from these nodes to the specified storage location. While processing, the nodes emit progress data to track their backup work. The coordinator node will aggregate the progress data into checkpoint files in the storage bucket. The checkpoint files provide a marker for the backup to resume after a retryable state, such as when it has been paused.
```

Comment:
> While processing, the nodes emit progress data to track their backup work. The coordinator node will aggregate the progress data into checkpoint files in the storage bucket. 
 
 How is this emitted? Do N1 and N3 send this to N2, and N2 aggregates it?

Reply:
Good point, that was slightly unclear. I think I have cleared this up and referred to the diagram here for the information.